mailpoet/lib/Automation/Engine/Storage/AutomationStorage.php

```diff
@@ -96,6 +96,35 @@ function($row): array {
     ) : [];
   }
 
+  /**
+   * @param int[] $versionIds
+   * @return Automation[]
+   */
+  public function getAutomationWithDifferentVersions(array $versionIds): array {
```

Comment:
Note: this method allows getting versions of different automations as it doesn't check all versions belong to the same automation (it does only in a test).

Reply:
Yes, that's true. Not sure, if that is a problem. But I am currently thinking if `getAutomationsByVersionIds()` is maybe a better name for this method :thinking: 

If you want, I can extend the method like 
```
getAutomationWithDifferentVersions(int $automationId, array $versionIds)
```

Reply:
The new method name you proposed would more accurately describe what the function does, but we can leave it as is, not big of a problem.